README.md

```diff
@@ -68,6 +68,8 @@ Join us on [Discord](https://astro.build/chat) to meet other maintainers. We'll
 | [@astrojs/tailwind](packages/integrations/tailwind)                                        | [![astro version](https://img.shields.io/npm/v/@astrojs/tailwind.svg?label=%20)](packages/integrations/tailwind/CHANGELOG.md)                                        |
 | [@astrojs/alpinejs](packages/integrations/alpinejs)                                        | [![astro version](https://img.shields.io/npm/v/@astrojs/alpinejs.svg?label=%20)](packages/integrations/alpinejs/CHANGELOG.md)                                        |
 | [@astrojs/mdx](packages/integrations/mdx)                                                  | [![astro version](https://img.shields.io/npm/v/@astrojs/mdx.svg?label=%20)](packages/integrations/mdx/CHANGELOG.md)                                                  |
+| [@astrojs/db](packages/db)                                                                 | [![create-astro version](https://img.shields.io/npm/v/@astrojs/db.svg?label=%20)](packages/db/CHANGELOG.md)                                                          |
+| [@astrojs/rss](packages/integrations/astro-rss)                                            | [![create-astro version](https://img.shields.io/npm/v/@astrojs/rss.svg?label=%20)](packages/astro-rss/CHANGELOG.md)                                                  |
```

Comment:
The image alt seems to be wrong (but it's also wrong for the other ones too), maybe we should fix them here together

Reply:
Copy paste skill issues 🤣

Thank you